helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -699,12 +699,12 @@ service:
       processors:
         - memory_limiter
         - k8sattributes
-        - batch
         - resourcedetection
         - resource
         {{- if .Values.environment }}
         - resource/add_environment
         {{- end }}
+        - batch
```

Comment:
Same. No need to change it here. Resource enrichment processors should work better on batches